Xamarin.Essentials/Platform/Platform.android.cs

```diff
@@ -106,8 +106,6 @@ public static void OnResume(Activity activity = null)
         {
             if (activity != null)
                 CheckAppActions(activity.Intent);
-
-            WebAuthenticator.OnResume(null);
```

Comment:
This i no longer used as the main activity neve has to handle the authentication events. This is now done by the intermediate activity.